apps/api/src/queries/dynamicForms/buildCommandForm.ts

```diff
@@ -14,6 +15,12 @@ import { buildFormFieldForCommandPayloadProp } from './buildFormFieldForCommandP
 // TODO We should have a SST for `rawData` as a key on all CREATE_X payloads.
```

Comment:
Revisiting this now. I think we may only want `rawData` on `IMPORT_X` commands, but let's not bother updating the comment for now.